src/cmd_app.rs

```diff
@@ -31,19 +32,48 @@ pub struct CmdApp {
 impl crate::cmd::Command for CmdApp {
     async fn run(&self, ctx: &mut crate::context::Context) -> Result<()> {
         let app_path = get_app_path()?;
+        let extra_args = get_extra_args()?;
 
         writeln!(ctx.io.out, "Opening the Zoo Design Studio at {}", app_path.display())?;
 
-        std::process::Command::new(app_path).arg(&self.path).spawn()?;
+        std::process::Command::new(app_path)
+            .arg(&self.path)
+            .args(&extra_args)
+            .spawn()?;
 
         Ok(())
     }
 }
 
 #[cfg(target_os = "linux")]
-/// Get the path to the application on linux.
+/// Get the path to the application on linux, assuming .AppImage installation as
+/// suggested at https://github.com/KittyCAD/modeling-app/blob/ac23d40e0bc756028d3933060c0c4377e7f6b6a3/INSTALL.md#linux.
+// TODO: consider other install locations
 fn get_app_path() -> Result<std::path::PathBuf> {
-    anyhow::bail!("We don't yet support Linux, but we are working on it!");
+    match dirs::home_dir() {
+        Some(home) => {
+            let path = home
+                .join("Applications")
```

Comment:
Likely not trivial at this point, started thread at https://kittycadworkspace.slack.com/archives/C04KFV6NKL0/p1758222119353819

Reply:
Guess this could be a * too 🫠